### PR TITLE
feat(jest): 100% coverage for store/groups/mutations

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

self-explanatory

**Which issue(s) this PR fixes** 🔨

no tickets at the moment

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by pavlzk — APPROVED** [review]

File: store/groups/mutations.test.ts [hunk 1]
```diff
@@ -191,3 +526,79 @@ describe('mutations.default.setGroupMembers', () => {
...
+      },
+    ]
+    mutations.default.setGroups(localState, argument)
+    expect(localState.all).toMatchObject(argument)
+  })
+  test('mutations.default.updateGroup', () => {
+    const localState = { ...GroupsInitialState }
+    const argument = {
+      id: '372d9ee6-4888-4280-90cd-a67d4c02a1c3',
+      // name: 'Sam Goldner', THE OLD ONE
```
Comment (stavares843) [inline store/groups/mutations.test.ts:572]: do we need this commented-out code?

File: store/groups/mutations.test.ts [hunk 2]
```diff
@@ -191,3 +526,79 @@ describe('mutations.default.setGroupMembers', () => {
...
+      name: 'Sam Gold', // Updated
+    }
+    mutations.default.updateGroup(localState, argument)
+    expect(localState.all[3].name).toContain(argument.name)
+  })
+  test('mutations.default.setGroupsLastUpdate', () => {
+    const localState = { ...GroupsInitialState }
+    const argument = {
+      '372d9ee6-4888-4280-90cd-a67d4c02a1c3': 1234, // Updated
```
Comment (stavares843) [inline store/groups/mutations.test.ts:581]: what does ''Updated'' mean in this specifically?

Comment (drepram) [reply store/groups/mutations.test.ts:581]: The ID of the group that we are trying to mutate. Shall I rename it `// Group that we are mutating, wrapped in this payload object`?

File: store/groups/mutations.test.ts (hunk 1 above)
Comment (drepram) [reply store/groups/mutations.test.ts:572]: I suppose, yes. To emphasize what we are changing. Do you want me to remove it?

File: store/groups/mutations.test.ts (hunk 2 above)
Comment (stavares843) [reply store/groups/mutations.test.ts:581]: yes please 🔨

File: store/groups/mutations.test.ts (hunk 1 above)
Comment (stavares843) [reply store/groups/mutations.test.ts:572]: yes please 🔨

Comment (drepram) [reply store/groups/mutations.test.ts]: Done.

File: store/groups/mutations.test.ts (hunk 2 above)
Comment (drepram) [reply store/groups/mutations.test.ts]: Done.

**Review by stavares843 — APPROVED** [review]

**Pull request merged**